src/features/machines/getTxMachine.ts

```diff
@@ -0,0 +1,28 @@
+import { getTx } from "src/services/nearHttpClient"
+
+export const getNearTxSuccessValue = async ({
+  txHash,
+  senderAccountId,
+}: {
+  txHash: string
+  senderAccountId: string
+}): Promise<bigint> => {
+  try {
+    const response = await getTx({
+      tx_hash: txHash,
+      sender_account_id: senderAccountId,
+      wait_until: "EXECUTED",
+    })
+    if (response.status?.SuccessValue) {
+      const decodedValue = Buffer.from(
+        response.status.SuccessValue,
+        "base64"
+      ).toString("utf-8")
+      // Parse the JSON string and convert to BigInt
+      return BigInt(JSON.parse(decodedValue))
```

Comment:
I don't think it is correct. Because you can't know for sure what kind of data inside value.

Reply:
Actually, we know because we understand the PRC method, which has a predefined response interface at the top level.